src/utils/getVaults.js

```diff
@@ -1,7 +1,6 @@
 const getVaults = async vaultsEndpoint => {
   try {
     let vaults = await fetch(vaultsEndpoint).then(res => res.json());
-    vaults = vaults.filter(vault => !vault.isGovVault);
     return vaults;
```

Comment:
Did you check how this affects every usage of `getVaults`, this function is also used from `getChainTvl` and it'd be getting new objects that aren't vaults.